packages/flutter/lib/src/material/material.dart

```diff
@@ -301,20 +316,43 @@ class Material extends StatefulWidget {
   static const double defaultSplashRadius = 35.0;
 }
 
+// Apply a semi-transparent white on surface colors to
+// indicate the level of elevation.
+Color _elevationOverlayColor(BuildContext context, Color background, double elevation) {
+  final ThemeData theme = Theme.of(context);
+  if (elevation > 0.0 &&
```

Comment:
Conventional formatting for this would be one-line or:
```
  if (elevation > 0.0 &&
      theme.applyElevationOverlay &&
      background == theme.colorScheme.surface) {
```